### ISSUE TITLE
Pagination component - remove aria-label from <div>s

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#1453]
Only certain html elements benefit from having an aria-label attribute. This attribute should be removed from the following `<div>`s of the Pagination component. 

- `<div class="pf-c-pagination" aria-label="Element pagination">`
- `<div class="pf-c-pagination__nav-page-select" aria-label="Current page 1 of 4">`

**Comment by srambach** [comment]
@jgiardino Just to doublecheck - this remains as is? 
`<nav class="pf-c-pagination__nav" aria-label="Pagination">`

**Comment by jgiardino** [comment]
Yes - `<nav>` elements are landmark roles, and can be labeled, and we definitely want the label there, since there will most likely be several `<nav>` landmarks on a page. 

This is the best article I've found so far that explains when you should use them: https://developer.paciellogroup.com/blog/2017/07/short-note-on-aria-label-aria-labelledby-and-aria-describedby/
